web/src/components/storage/SpacePolicySelection.tsx

```diff
@@ -131,10 +119,10 @@ export default function SpacePolicySelection() {
 
   // Resets actions (i.e., sets everything to "keep") if the custom policy has not been used yet.
   useEffect(() => {
-    if (policy?.id !== "custom" && !customUsed) setActions([]);
+    if (policy && policy?.id !== "custom" && !customUsed) setActions([]);
```

Comment:
Nice catch!

Reply:
Actually, I think I can remove the `?`.

Reply:
Yes you can. My brain deleted it automatically when reading it a couple of minutes ago.